src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
According to the checklist, you should not create a new `FigureSupplier` instance inside a loop. However, there is no loop here for creating figures. Ensure that the `FigureSupplier` class does not create a new instance within a loop when generating figures.